### PR TITLE
test/vlan: 3-level VLAN test

### PR DESCRIPTION
**Pull request by jlucovsky**
Issue: 2816

This commit tests for 3-level VLAN encapsulations

The Suricata PR: https://github.com/OISF/suricata/pull/7786

### CODE REVIEW

**Comment by catenacyber** [comment]
cf https://github.com/OISF/suricata/pull/7700

I think this would require a min-version 7...

**Comment by jlucovsky** [comment]
> cf [OISF/suricata#7700](https://github.com/OISF/suricata/pull/7700)
> 
> I think this would require a min-version 7...

Good point -- will update.

File: tests/eve-flow-vlan-02/test.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
```
Comment (catenacyber) [inline tests/eve-flow-vlan-02/test.yaml:3]: `HAVE_LIBJANSSON` is implied by `min-version: 7` ;-)

Comment (jlucovsky) [reply tests/eve-flow-vlan-02/test.yaml:3]: :-)

**Comment by catenacyber** [comment]
Rebased in https://github.com/OISF/suricata-verify/pull/1204

**Pull request closed without merging**